src/commands/deploy/genezio.ts

```diff
@@ -207,6 +211,25 @@ export async function genezioDeploy(options: GenezioDeployOptions) {
             throw error;
         });
 
+        // Handle Python requirements.txt file
+        if (configuration.backend.language.name === Language.python) {
+            const requirementsPath = path.join(backendCwd, "requirements.txt");
+
+            if (fs.existsSync(requirementsPath)) {
```

Comment:
i don't really like this logic being here, the `genezioDeploy` function should not have any language specific logic. is there a way you could include this step in the python bundler?

Reply:
also, don't think we should modify the project structure of our users code. if you want to create a folder containing requirements, do this in the temporary folder created in the next steps (I will leave a comment there also)